src/pipeline-server/docker-compose.pipeline-server.yml

```diff
@@ -190,28 +172,16 @@ services:
           memory: 1024M
       restart_policy: 
         condition: on-failure
-        max_attempts: 5          
+        max_attempts: 5
     volumes:
        - "vol_temp_grafana:/tmp"
        - ${RETAIL_USE_CASE_ROOT:-..}/src/pipeline-server/grafana:/app/eva
-    environment:
-      HOST_IP: ${HOST_IP}
-      MODE: "EVA"
-      http_proxy: ${http_proxy}
-      https_proxy: ${https_proxy}
-      no_proxy: ${no_proxy}
```

Comment:
Same as above, it is recommended to take the proxy settings from the host system.